plugins/inputs/procstat/procstat_test.go

```diff
@@ -369,3 +370,39 @@ func TestGather_cgroupPIDs(t *testing.T) {
 	assert.Equal(t, []PID{1234, 5678}, pids)
 	assert.Equal(t, td, tags["cgroup"])
 }
+
+func TestExitStatus(t *testing.T) {
+	out, err := exec.Command("pgrep", "-G", "sys").Output()
```

Comment:
This command could return different results on different systems, so it could create random failures, perhaps you can make the needed error by hand.  Also, it tests a function in `internal/internal.go` so it should be placed in `internal/internal_test.go`